src/main/java/com/minecolonies/coremod/inventory/api/package-info.java

```diff
@@ -0,0 +1,5 @@
+/**
+ * Created by marcf on 3/25/2017.
+ */
+package com.minecolonies.coremod.inventory.api;
+//TODO: Move this package to its own API-Module.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)